nano/templates/shipsensors.tmpl

```diff
@@ -128,6 +128,16 @@
 	</tr>
 	{{/for}}
 	</table>
+  {{if data.contact_details}}
+	<class="itemContent">
+	{{:data.contact_details}}
+	<br/>
+	</div>
+	<class="item">
+	{{:helper.link('Print', 'search', { 'scan' : 'print' })}}
+	{{:helper.link('Clear', 'refresh', { 'scan' : 'clear' })}}
```

Comment:
I am not really a fan of re-using the scan parameter here with "special" values for print/clear.
Instead that should be separate parameters.